packages/manager/apps/hub/src/App.tsx

```diff
@@ -21,7 +22,7 @@ function App() {
   const router = createHashRouter(Routes);
 
   useEffect(() => {
-    shell.ux.hidePreloader();
+    defineCurrentPage(`app.dashboard`);
```

Comment:
**nitpick:** This can be done in more generic way to handle page tracking and request tagger in an automated way. No need to do it specific for a given page.